pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -412,65 +479,16 @@ func (o *MustGatherOptions) Run() error {
 	defer o.logTimestamp()
 
 	var wg sync.WaitGroup
-	wg.Add(len(pods))
 	errCh := make(chan error, len(pods))
+	podChan := make(chan *corev1.Pod)
+	wg.Add(concurrentMG)
```

Comment:
`wg` is no longer needed.

Reply:
Why? How do you suggest we wait for the goroutines to complete in that case?

Reply:
You are right. Please ignore my comment.

Reply:
Ack, marking resolved.